setup.py

```diff
@@ -35,11 +35,18 @@
         ]
     },
     install_requires=[
+        'pillow',
         'pandas',
         'pymongo',
         'pyserial',
         'six',
-        'utm'
+        'scipy',
+        'utm',
+        'matplotlib',
+        'jupyter',
+        'ipyleaflet',
```

Comment:
Can `ipyleaflet` be installed this way?  I thought you needed to also [run something like the following:](https://github.com/jupyter-widgets/ipyleaflet)
```
$ pip install ipyleaflet
$ jupyter nbextension enable --py --sys-prefix ipyleaflet  # can be skipped for
notebook 5.3 and above
```